### PR TITLE
Add README section about executable installation using Chocolatey

### PR DESCRIPTION
**Pull request by tadeu**
I've created a project ([clcache-builder](https://github.com/ESSS/clcache-builder)) to build an executable version of `clcache.py` and to package it as a Chocolatey package, in order to make it easier to use `clcache` on environments without Python or with Python 2.

Please, feel free to review that project too. If you are interested in adding the package building to this repository, I could give some help,

### CODE REVIEW

**Comment by tadeu** [comment]
cc @nicoddemus

File: README.asciidoc [hunk 1]
```diff
@@ -57,6 +57,22 @@ way and replaced with the executable generated by PyInstaller:
...
+Installation of executable via Chocolatey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a very easy installation method, and useful when needing to use clcache in environments with Python 2 (or even without Python), since clcache supports only Python 3.3+.
+
+Just install https://chocolatey.org/[Chocolatey] and run
+
+    choco install clcache
+
+The package is automatically generated using AppVeyor by the https://github.com/ESSS/clcache-builder[clcache-builder project].
```
Comment (nicoddemus) [inline README.asciidoc]: This link formatting is wrong, I think it should be:

```markdown
The package is automatically generated using AppVeyor by [ESSS/clcache-builder](https://github.com/ESSS/clcache-builder).
```

Comment (nicoddemus) [inline README.asciidoc]: Oops sorry, disregard my comment, I thought this was Markdown. 😝

**Comment by frerich** [comment]
Cool stuff! I'll give it a try -- as for package building: any help with that would be much appreciated. I've planned to provide packages of clcache (e.g. ZIP files with the binaries produced by PyInstaller) for a long time but never got around to learning how to do that. I just saw that AppVeyor (which we use for CI) supports 'deploying' data and GitHub supports hosting binaries for releases. A Chocolatey package might be a nice addition (or a start)! :-)

**Comment by tadeu** [comment]
@frerich You could try merging the steps of https://github.com/ESSS/clcache-builder/blob/master/appveyor.yml into your `appveyor.yml`, and make a [nuspec file](https://github.com/ESSS/clcache-builder/blob/master/clcache.nuspec) based on the version there. After setting these things up, it's pretty straightforward, it comes down to `pip install pyinstaller`, `pyinstaller clcache.py` and `choco pack`, the other stuff is version and artifact management.

**Comment by tadeu** [comment]
Chocolatey have a very nice feature that allows only `clcache.exe` to stay in the PATH, the other `.dll` and `.pyd` stay in a directory that is not in the PATH, so won't cause problems with executables that depend on libraries with the same name.

**Comment by frerich** [comment]
@tadeu Thanks a lot for your awesome work on packaging clcache! This has always been a bit of a blind spot for me - I hope you don't mind me taking your set of packages to define a first [clcache release](https://github.com/frerich/clcache/releases) in the GitHub sense. :-)

As for this pull request: I'm considering to not extending the README file even further (it's fairly long already) but rather making more use of the [Wiki](https://github.com/frerich/clcache/wiki). There are already a few initial pages and I think it might be good to have an 'Installation' page which then references an 'Installation via Chocolatey' page. The benefit of this is that I could reference it nicely from other places. What do you think?

**Comment by tadeu** [comment]
@frerich Thanks :)

I agree, you may drop these modifications here, since some of them don't make sense anymore (the build is already being done in this repo), and using the Wiki seems good. It would be nice to have a link in the README pointing to the Wiki, and perhaps other sections could be moved too.

**Comment by frerich** [comment]
Closing this ticket - I recently revamped the Wiki and added the instructions for [how to use Chocolatey](https://github.com/frerich/clcache/wiki/Building-clcache#installing-an-executable-file-via-chocolatey) there.

**Pull request closed without merging**